### DISCUSSION TITLE
新增文章如何在代码中预置其他参数

### DISCUSSION THREAD

**Discussion by CYY118** [Qexo/Qexo#571] [Q&A] [answered]
![Screenshot_20240905_204604_edit_537356010369024.jpg](https://github.com/user-attachments/assets/628e74cc-d583-451e-abfd-253cb09e65a7)

如何在相关代码中把我想新建文章时的参数设置好 不用每次新建文章都都去添加参数

**Comment by am-abudu** [comment, marked as answer]
修改文章模板

**Reply by CYY118** [reply]
你好  具体是修改哪里

**Reply by am-abudu** [reply]
如果是Hexo，修改scaffold

See: [https://hexo.io/zh-cn/docs/writing#Scaffolds](https://hexo.io/zh-cn/docs/writing#Scaffolds)

**Reply by CYY118** [reply]
解决了  非常感谢
